specs/stuck.spec.js

```diff
@@ -51,9 +51,11 @@ describe('Stuck', () => {
       }
     `});
     await page.addScriptTag({ path: 'lib/index.js' });
-  });
+    await page.waitFor(60);
```

Comment:
👀 This is workaround to run tests after puppeteer's rendering finished. I think there are other ways to avoid this.